xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp

```diff
@@ -127,14 +128,18 @@ bool CGUIDialogPVRGuideInfo::OnClickButtonPlay(CGUIMessage &message)
 {
   bool bReturn = false;
 
-  if (message.GetSenderId() == CONTROL_BTN_SWITCH || message.GetSenderId() == CONTROL_BTN_PLAY_RECORDING)
+  if (message.GetSenderId() == CONTROL_BTN_SWITCH || message.GetSenderId() == CONTROL_BTN_PLAY_RECORDING || message.GetSenderId() == CONTROL_BTN_PLAY_TAG)
   {
     Close();
 
     if (m_progItem)
     {
       if (message.GetSenderId() == CONTROL_BTN_PLAY_RECORDING && m_progItem->HasRecording())
         g_application.PlayFile(CFileItem(m_progItem->Recording()), "videoplayer");
+      if (message.GetSenderId() == CONTROL_BTN_PLAY_TAG && m_progItem->IsPlayable()) {
+        m_progItem->SetPath(m_progItem->GetStreamUrl());
+        g_application.PlayFile(CFileItem(m_progItem), "videoplayer");
```

Comment:
Why is this functionality only available from the guide dialog, not from the guide window?